sigopt_sklearn/ensemble.py

```diff
@@ -47,26 +52,25 @@ def find_bayes_avg_coefs(self, X, y):
     z = z / np.sum(z)
     return z
 
-  def fit(self, X, y, client_token=None, cv=5, est_timeout=None,
-          cv_timeout=None, n_iter=25, n_jobs=1, n_cv_jobs=5):
+  def fit(self, X, y, client_token=None, est_timeout=None):
     self.n_outputs_ = 1
-    self.classes_ = np.unique(check_array(y, ensure_2d=False,
-                                          allow_nd=True, dtype=None))
-    #Store X and y data for workers to use
+    self.classes_ = np.array(np.unique(check_array(y, ensure_2d=False,
```

Comment:
Hm, linter was complaining that it can return other values. This might be extraneous then